edge_computing/day_2_core_cnf_clusters/updating/update-completing-the-control-plane-only-update.adoc

```diff
@@ -0,0 +1,69 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="update-completing-the-update"]
+= Completing the Control Plane Only cluster update
```

Comment:
There's a later section called "Updating the worker nodes", so I'd rename this "Updating the control plane nodes" for parallelism